static/index.html

```diff
@@ -84,20 +82,35 @@
       }
 
       function create(encounterId) {
-        $.post('/hangouts', {encounterId: encounterId}, (data, status) => {
+        $.post('/hangouts', { encounterId: encounterId }, (data, status) => {
           if (data['url']) {
-            window.location.replace(data['url']);
+            const url = data['url'];
+            if (url.includes('meet.google.com')) {
+              showJoinButton(url);
+            } else {
+              window.location.replace(url);
```

Comment:
When will the URL not include meet.google.com?

Reply:
/authenticate url which is needed to create a calendar event.

Reply:
Can you add a comment?

Reply:
done